### PR TITLE
Split downloads for unstable builds into separate file

### PR DESCRIPTION
**Pull request by mathiascode**
Not finished yet, but this is something I had in mind to keep our number of doc files small and separate the unstable downloads from stable.

@hboetes

### CODE REVIEW

**Comment by hboetes** [comment]
Yes! Good thinking!

File: doc/DOWNLOADS.md [hunk 1]
```diff
@@ -18,77 +20,53 @@ sudo apt install nicotine
...
 ```
 
-### Void Linux (Stable)
+### Void Linux
 
 To install Nicotine+ on Void Linux, run the following:
 
 ```sh
 sudo xbps-install -S nicotine+
 ```
```
Comment (hboetes) [inline doc/DOWNLOADS.md:52]: ### FreeBSD

To install Nicotine+ on FreeBSD, run the following:

```sh
pkg install py-nicotine-plus
```

File: doc/DOWNLOADS.md [hunk 2]
```diff
@@ -18,77 +20,53 @@ sudo apt install nicotine
...
-```
-
-### Other Distributions
-
-If Nicotine+ has not been packaged for your distribution/operating system yet, there are other recommended ways of installing Nicotine+.
-
-#### pip (Stable)
+### pip
 
 Nicotine+ can be installed using [pip](https://pip.pypa.io/en/stable/). Ensure the [runtime dependencies](doc/DEPENDENCIES.md) are installed, and run the following:
```
Comment (hboetes) [inline doc/DOWNLOADS.md:55]: I'd use this construction:

> For all operating systems where no official package is provided, we recommend installing Nicotine+ using pip. If you don't already have pip installed, it's most certainly available in your packaging system. Ensure the [runtime....

**Review by hboetes — COMMENTED** [review]
Minor nits.

File: doc/DOWNLOADS.md (hunk 1 above)
Comment (mathiascode) [reply doc/DOWNLOADS.md:52]: It was added in https://github.com/nicotine-plus/nicotine-plus/commit/29d9fdb9148ca536985bf018e4dbc434f5cd54f5

**Pull request merged**